### PR TITLE
feat: manual fee management, admin account management

### PR DESCRIPTION
**Pull request by sunmoon11100**

### CODE REVIEW

**Review by borispovod — CHANGES_REQUESTED** [review]
We need tests for functions not covered, maybe tests to check how swap works with min/max fees values, etc.

And we need to use resource account for emergency.

File: sources/swap/liquidity_pool.move [hunk 1]
```diff
@@ -660,9 +676,58 @@ module liquidswap::liquidity_pool {
...
+        let pool = borrow_global<LiquidityPool<X, Y, Curve>>(@liquidswap_pool_account);
+        pool.dao_fee
+    }
+
+    /// Set DAO fee for specific pool.
+    public entry fun set_dao_fee<X, Y, Curve>(fee_admin: &signer, dao_fee: u64) acquires LiquidityPool {
+        assert!(coin_helper::is_sorted<X, Y>(), ERR_WRONG_PAIR_ORDERING);
+        assert!(exists<LiquidityPool<X, Y, Curve>>(@liquidswap_pool_account), ERR_POOL_DOES_NOT_EXIST);
+        assert_pool_unlocked<X, Y, Curve>();
+        assert!(signer::address_of(fee_admin) == global_config::get_fee_admin(), ERR_NOT_ADMIN);
```
Comment (sunmoon11100) [inline sources/swap/liquidity_pool.move:725]: Is it correct that the fee admin can set the dao fee and not the dao admin?

Comment (borispovod) [reply sources/swap/liquidity_pool.move:725]: Correct, dao only withdraws rewards for now.

Comment (mkurnikov) [reply sources/swap/liquidity_pool.move:725]: @borispovod

**Review by borispovod — APPROVED** [review]
LGTM

**Review by mkurnikov — APPROVED** [review]

**Pull request merged**